perf/perf_bench.py

```diff
@@ -66,10 +66,14 @@ def verify_dmesg(self):
                 self.fail("Test Failed : %s in dmesg" % fail_pattern)
 
     def run_cmd(self, cmd):
+        err_ln = "Assertion `!(ret)' failed"
```

Comment:
is this exact string we are seeing in assertion case if not i feel we can use regex pattern for same

Reply:
@PraveenPenguin yes this is the exact string we see

```
# Running  1x3-convergence, "perf bench numa mem -p 1 -t 3 -P 512 -s 100 -zZ0qcm --thp  1"
perf: bench/numa.c:302: bind_to_node: Assertion `!(ret)' failed.
Aborted (core dumped)

# Running main, "perf bench numa numa-mem -p 2 -t 1 -P 1024 -C 0,8 -M 1,0 -s 20 -zZq --thp 1 --no-data_rand_walk"
# NOTE: ignoring bind NODEs starting at NODE#1
# NOTE: 0 tasks mem-bound, 2 tasks unbound
perf: bench/numa.c:334: bind_to_cpumask: Assertion `!(ret)' failed.
```